xcode.md

```diff
@@ -159,8 +159,10 @@ Got access to old versions of Apple installers you know are legit? Submit some h
 
 | SHA1 | Version | Filename |
 | ------- | ---- | -------- |
+| `f1aeb8cc3cc0092e3fcd569f1bd9234d9c25d666` | [CLI Tools 9.3 beta (10.13)][cli-9.3b1-10.13] | Command_Line_Tools_macOS_10.13_for_Xcode_9.3_beta.dmg
```

Comment:
If you’re going to do this, please include a <!-- commented out --> SHA-256 for any new hashes too, as you did for the 9.3 beta above.

Reply:
Not an issue... more hashes coming soon!